explore/src/main/scala/explore/config/TimeAndCountEditor.scala

```diff
@@ -43,19 +43,14 @@ object TimeAndCountEditor extends ConfigurationFormats:
       val count           = props.options.zoom(TimeAndCountModeInfo.count)
       val signalToNoiseAt = props.options.zoom(TimeAndCountModeInfo.at)
 
-      val timeFormat = props.instrument
+      val (timeFormat, timeAuditor) = props.instrument
         .map {
-          case Instrument.GmosSouth | Instrument.GmosNorth => durationS.optional
-          case _                                           => durationMs.optional
+          case Instrument.GmosSouth | Instrument.GmosNorth | Instrument.Flamingos2 =>
+            (durationS.optional, ChangeAuditor.int.optional)
+          case _                                                                   =>
+            (durationMs.optional, ChangeAuditor.posBigDecimal(2.refined).optional)
```

Comment:
I see that the `_` case is the same as the default. I think  we `collect` instead of `map` and avoid the `_` clause.

Reply:
good, I'll add it

Reply:
hmm collect returns `Option[B]`. it makes it more complicated I think